cmd/kubepkg/templates/latest/rpm/cri-tools/cri-tools.spec

```diff
@@ -19,7 +19,7 @@ Command-line utility for interacting with a container runtime.
 # TODO: Do we need these?
 #%autosetup
 #%build
-#%configure
+#%%configure
```

Comment:
this seems fine as per:
http://ftp.rpm.org/max-rpm/ch-rpm-specref.html

but the question is do we want to expose `./configure` for the spec even.
is it really needed?

one can still do `rpmbuild....specfile`.